### ISSUE TITLE
Hide OpenAPI documentation for non-admin users

### ISSUE THREAD

**Issue by mingsai** [mevdschee/php-crud-api#671]
> The "/openapi" endpoint will only show what is allowed in your session. It also has a special "document" operation to allow you to hide tables and columns from the documentation.

**UPDATE:**
Added the following to my authorization.tableHandler. Anyway to remove the default text remaining so that no information at all is shown? For admin users I'd modify the criteria.
```
  if ($operation == 'document'){
            return '';
   }
```

I want to disallow or hide the openapi documentation for anyone except my admin users. How can I accomplish that?

You mention a 'document' operation. I've taken a look at the OpenAPI Controller but it seems like this might be something to handle in a handler. Not sure if this would integrate with a tableHandler or if it could be better placed elsewhere.

Can you provide a sample usage?

**Comment by mevdschee** [comment]
Did you try this?

    return false;

I'm not sure..

**Comment by mingsai** [comment]
Tested this option in the config section but it still showed the info section (not the tables when using the tableHandler as noted above)

      '`openApiBase' => '{}' ,`

I did try returning false from other handlers but I found that the other handlers did not further close off the info, but the config area allows me to override the controllers.

Adding this to the config section and changing from:

    `  'controllers' => 'records,geojson,openapi', `

To:

     ` 'controllers' => 'records,geojson', `

Disables the OpenAPI route totally, while using the tableHandler allows only admins to see tabular information. Could not find an override that works for hiding the system info from non-admins only. I will just disable the feature for now.
